pkg/flowkit/config/json/config.go

```diff
@@ -129,9 +130,8 @@ func (p *Parser) Deserialize(raw []byte) (*config.Config, error) {
 
 	var jsonConf jsonConfig
 	err := json.Unmarshal(raw, &jsonConf)
-
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("configuration syntax error: %v", err)
```

Comment:
`%w` is probably preferable to `%v` here  
```suggestion
		return nil, fmt.Errorf("configuration syntax error: %w", err)
```